src/pages/new/index.js

```diff
@@ -154,6 +157,7 @@ export default () => {
           },
         ]}
       />
+      <Spacer heightDesktop="20px" heightMobile="0" />
```

Comment:
It seems like there's an extra 10px of margin being caused by the bottom of the carousel dots. Should we correct that in that component?

Also, shouldn't the mobile height be set to 40px here, to combine with the 40px of padding in the Partners and Sponsors block in order to get a total of 80px between the carousel and the Partners and Sponsors block?

Reply:
setting heightDesktop to 0 and heightMobile to 40px

Reply:
Ill make tickets for the subtle corrections

Reply:
https://app.clubhouse.io/sheltertech/story/1748/fix-spacing-in-carousel